panic if I don't have correct rights

Hi there, I found the problem in case if I do Get for id, where I don't have (full) rights.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x11b12a8]

goroutine 1 [running]:
github.com/cvbarros/go-teamcity/teamcity.(*Parameters).NonInherited(0x0, 0xc000306aa0)
	/home/seidl/GoWorkspace/pkg/mod/github.com/cvbarros/go-teamcity@v1.1.0/teamcity/parameter.go:187 +0x78
github.com/cvbarros/go-teamcity/teamcity.(*ProjectService).GetByID(0xc0004fc040, 0xc00004a730, 0xd, 0x1, 0x1, 0xe)
	/home/seidl/GoWorkspace/pkg/mod/github.com/cvbarros/go-teamcity@v1.1.0/teamcity/project.go:123 +0x112
main.Teamcity(0xc000147ba0, 0x8b000000014053c0)
```

the API response looks like
```
{
    "id": "bbc",
    "name": "BBC",
    "parentProjectId": "_Root",
    "href": "/httpAuth/app/rest/projects/id:bbc",
    "webUrl": "https://teamcity/project.html?projectId=bbc",
    "parentProject": {
        "id": "_Root",
        "name": "<Root project>",
        "description": "Contains all other projects",
        "href": "/httpAuth/app/rest/projects/id:_Root",
        "webUrl": "https://teamcity/project.html?projectId=_Root"
    },
    "buildTypes": {
        "count": 2,
        "buildType": [
            {
                ...
            },
            {
                ...
            }
        ]
    },
    "projects": {
        "count": 0
    }
}
```

From my point of view, it is a bug. The one option is to get `Project` struct without that information, second is do `error` before call method on (in this case non-exists) `Parameters` https://github.com/cvbarros/go-teamcity/blob/master/teamcity/project.go#L123

--- a/teamcity/project.go
+++ b/teamcity/project.go
@@ -120,7 +120,10 @@ func (s *ProjectService) GetByID(id string) (*Project, error) {
 	}
 
 	//For now, filter all inherited parameters, until figuring out a proper way of exposing filtering options to the caller
-	out.Parameters = out.Parameters.NonInherited()
+	//Parameters can be nil for users that have no permission to view them
+	if out.Parameters != nil {
+		out.Parameters = out.Parameters.NonInherited()
+	}
 	return &out, err
 }
 

